locales/en/translation.json

```diff
@@ -336,5 +336,12 @@
   "{{date}} - {{time}}": "{{date}} - {{time}}",
   "{{numVisibleDays}} days in view": "{{numVisibleDays}} days in view",
   "{{patientName}} does not have any data yet.": "{{patientName}} does not have any data yet.",
-  "{{title}} (cont.)": "{{title}} (cont.)"
+  "{{title}} (cont.)": "{{title}} (cont.)",
```

Comment:
Can you sort these value alphabetically in the file?

Reply:
Also the keys do not look like what we are used to :thinking: 
Shouldn't it be using this format: `toto-tata`?